wombase_backend/server/apps/tools/views.py

```diff
@@ -50,51 +63,55 @@ class ToolTransferAPIView(generics.UpdateAPIView):
     def patch(self, request, *args, **kwargs):
         tool = self.get_object()
         data = request.data
-        if data.get("owner") is not None and data.get("currently_at") is not None:
+        owner = data.get("owner")
```

Comment:
maybe rename this variable to "data_owner" or "request_owner" etc to make more readable variable